src/test/java/org/alfresco/rest/api/tests/client/PublicApiClient.java

```diff
@@ -847,17 +849,82 @@ public JSONObject parseListSource(JSONObject jsonResponse)
             assertNotNull(source);
             return source;
         }
-
+        
+        /**
+         * Used for validation of status code on rest response
+         * 
+         * @param errorMessage

```

Comment:
Please remove empty tags as they are warnings in Javadoc and provide zero value: javadoc knows about the arguments and return from the method signature.